blocks/subscriptions-block/components/ContactInfo/index.jsx

```diff
@@ -0,0 +1,177 @@
+  };
+
+  const handleInputChange = (name, entry) => {
+    entriesRef.current[name] = entry.value;
+  };
+
+  const handleLogout = () => {
+    logoutCallback();
+  };
+
+  const { arcSite } = useFusionContext();
+  const phrases = getTranslatedPhrases(getProperties(arcSite).locale || 'en');
+
+  return (
+    <form onSubmit={handleSubmit} ref={formRef} className="xpmedia-subscriptions-contact-info">
+      <PrimaryFont as="h2" className="xpmedia-subscriptions-contact-info-title">
+        {phrases.t('checkout-block.contact-info')}
+      </PrimaryFont>
+
+      <FormInputField
+        autoComplete="email"
+        label={phrases.t('checkout-block.email')}
+        name="email"
+        defaultValue={user && user.email ? user.email : ''}
+        hidden={!!(user && user.email)}
+        required
+        onChange={(value) => { handleInputChange('email', value); }}
+        showDefaultError={false}
+        type={FIELD_TYPES.EMAIL}
+        validationErrorMessage={phrases.t('checkout-block.email-requirements')}
+      />
+      {user && user.email && (
+        <div className="identity-row">
+            {user?.identities?.[0]?.type && (
+              <>
+                {user.identities[0].type.toLowerCase() === 'google' && (
+                  <>
+                    <span className="identity-icon">
+                      <GoogleIcon />
+                    </span>
+                    <span>
+                      {phrases.t('checkout-block.identity-social')}
+                      {' '}
+                      <b>{`${user.email}`}</b>
+                    </span>
+                    <button
+                      type="button"
+                      className="sign-out-btn"
+                      onClick={handleLogout}
+                    >
+                      {phrases.t('checkout-block.identity-sign-out')}
+                    </button>
+                  </>
+                )}
+                {user.identities[0].type.toLowerCase() === 'facebook' && (
+                  <>
+                    <span className="identity-icon">
+                      <FacebookIcon />
+                    </span>
+                    <span>
+                      {phrases.t('checkout-block.identity-social')}
+                      {' '}
+                      <b>{`${user.email}`}</b>
+                    </span>
+                    <button
+                      type="button"
+                      className="sign-out-btn"
+                      onClick={handleLogout}
+                    >
+                      {phrases.t('checkout-block.identity-sign-out')}
+                    </button>
+                  </>
+                )}
+                {user.identities[0].type.toLowerCase() === 'password' && (
+                  <>
+                    <span>
+                      {phrases.t('checkout-block.identity-email')}
+                      {' '}
+                      <b>{user.email}</b>
+                    </span>
+                    <button
```

Comment:
This button is repeated three times - any way to only use one?

Reply:
Yup!